src/server/components/dashboard/listsItems/itemsRouter.ts

```diff
@@ -89,11 +89,17 @@ listRouter.post("/:listId/items/:listItemId", controllers.listItemPostController
 /**
  * TODO: ref to /:listItemId/:action?
  */
+
+// No moveUpdatedDataToListItem middleware needed for pin action
+listRouter.post("/:listId/items/:listItemId/pin", controllers.listItemPinController);
+
+listRouter.post("/:listId/items/:listItemId/*", controllers.moveUpdatedDataToListItem);
```

Comment:
it looks like on any action, the update is added to the jsonData immediately. They need to be approved before doing so. jsonData is used to render the details in the `find` journey, so we need to make sure we don't show anything that's not approved by post. 

Take for example a list item that currently has a pending change `json.updatedJsonData` is populated
- post realise another change has to be made, separate from their first change request
- so they request a change `/change` endpoint 

since the post endpoint is wild carded (*), `controllers.moveUpdatedDataToListItem` will run. 

the updatedJsonData hasn't been approved, but it's been committed to the jsonData - which is visible to the public. 

There is also a specific way to update jsonData - it needs an audit event. see `handleListItemUpdate`